src/api/app/views/webui/shared/user_or_groups_roles/_index.html.haml

```diff
@@ -61,7 +55,7 @@
                  locals: { records: groups, roles: roles, type: 'group',
                            user_is_maintainer: user_is_maintainer,
                            project: project, package: package }
-  - if !feature_enabled?(:responsive_ux) && user_is_maintainer
+  - if user_is_maintainer
     %ul.list-inline
```

Comment:
@dmarcoux the PR still contains the `nav-link` class and the inline list

Reply:
So we're using `.pt-4` now?

Reply:
Yes

Reply:
Please have another look

Reply:
Great